packages/cli/lib/validators/marketplaceValidators/theme/ThemeValidator.js

```diff
@@ -55,11 +66,25 @@ class ThemeConfigValidator extends BaseValidator {
             meta: { file: themeJSONFile },
           });
         }
+        if (!themeJSON.screenshot_path) {
+          validationErrors.push({
+            ...this.getError(this.errors.MISSING_SCREENSHOT_PATH),
+            meta: { file: themeJSONFile },
+          });
+        } else if (!isRelativePath(themeJSON.screenshot_path)) {
+          validationErrors.push({
+            ...this.getError(this.errors.ABSOLUTE_SCREENSHOT_PATH),
+            meta: { file: themeJSONFile },
+          });
+        }
```

Comment:
Not a blocker, but should we make sure that the file also exists?

Reply:
That's a good call. I'm not currently doing that, but I probably should. @KatzMitch is the current BE validation checking that the screenshot path points to an existing file?

Reply:
Ooh that's a really good call - we don't check that now but _really_ should

Reply:
Sweet, I can add in a check for that 👍

Reply:
Added support for this in 16cf83d